### ISSUE TITLE
Every year has different DST start and end

### ISSUE THREAD

**Issue by pkarlik** [serverless/serverless-local-schedule#11]
How this could work reliably considering every year has different DST start and end? At least this is how it works for CET and NZST.

**Comment by dschep** [comment]
Yeah. that's a limitation that I should document. You still have to deploy at least once a year, but it's still useful bc you dpn't have to worry about deploying explicitly bc DST changed over.

**Comment by dschep** [comment]
It _could_ generate a set of schedules that will last multiple years, but that probably consumes more formulation resources than most people would be willing to sacrifice (6 per year)
